src/chainparams.cpp

```diff
@@ -300,10 +300,11 @@ class CMainParams : public CChainParams {
             (1200000, uint256S("0x0000000000347d5011108fdcf667c93e622e8635c94e586556898e41db18d192"))
             (1400000, uint256S("0x0000000001155ecec0ad3924d47ad476c0a5ed7527b8776f53cbda1a780b9f76"))
             (1600000, uint256S("0x0000000000aae69fb228f90e77f34c24b7920667eaca726c3a3939536f03dcfc"))
-            (1860000, uint256S("0x000000000043a968c78af5fb8133e00e6fe340051c19dd969e53ab62bf3dc22a")),
-            1667052073,     // * UNIX timestamp of last checkpoint block
-            11768038,       // * total number of transactions between genesis and last checkpoint
-            3644            // * estimated number of transactions per day after checkpoint
+            (1860000, uint256S("0x000000000043a968c78af5fb8133e00e6fe340051c19dd969e53ab62bf3dc22a"))
+            (2000000, uint256S("0x00000000010accaf2f87934765dc2e0bf4823a2b1ae2c1395b334acfce52ad68")),
+            1677602242,     // * UNIX timestamp of last checkpoint block
+            12380742,       // * total number of transactions between genesis and last checkpoint
+            3566            // * estimated number of transactions per day after checkpoint
                             //   total number of tx / (checkpoint block height / (24 * 24))
```

Comment:
I just noticed that this calculation comment wasn't updated to account for Blossom activation. The calculation we want is:
$$\frac{\mathsf{totalChainTx}}{\mathsf{totalChainBlocks}} \cdot \mathsf{blocksPerDay} = \frac{\mathsf{tx}}{\mathsf{block}} \cdot \frac{\mathsf{block}}{\mathsf{day}} = \frac{\mathsf{tx}}{\mathsf{day}}$$



From this, we can determine that the calculation in the comment is:
$$\frac{\mathsf{totalChainTx}}{\frac{\mathsf{totalChainBlocks}}{\mathsf{blocksPerHour \cdot \mathsf{hoursPerDay}}}} = \frac{\mathsf{tx}}{\frac{\mathsf{block}}{(\mathsf{block} / \mathsf{hour}) \cdot (\mathsf{hour} / \mathsf{day})}} = \frac{\mathsf{tx}}{\frac{\mathsf{block}}{\mathsf{block} / \mathsf{day}}} = \frac{\mathsf{tx}}{\mathsf{block} \cdot \frac{\mathsf{day}}{\mathsf{block}}} = \frac{\mathsf{tx}}{\mathsf{day}}$$

The comment calculation is therefore using a value of 24 blocks per hour, but from the Blossom NU it doubled to 48 blocks per hour. So we should update this to:
```suggestion
            7131            // * estimated number of transactions per day after checkpoint
                            //   (total number of tx * 48 * 24) / checkpoint block height
```